### ISSUE TITLE
increment number bug

### ISSUE THREAD

**Issue by vTNT** [Voronenko/gitflow-release#2]
when VERSION is '0.9.0', i run bump-version-drynext.sh.  the next version is become 1.0.0, not 0.10.0;
also.  when VERSION is '0.0.9', run bump-minorversion-drynext.sh .  the next version is become 0.1.0, not 0.0.10

**Comment by Voronenko** [comment]
Confirmed. Feel free to open PR meanwhile

**Comment by Voronenko** [comment]
There are some good candidates  https://stackoverflow.com/questions/8653126/how-to-increment-version-number-in-a-shell-script  , like  Stephen M. Harris response. 

If you get this incorporated and tested, you are welcome.

**Comment by vTNT** [comment]
yes. Stephen M. Harris ‘s comment can solve my problem.  thanks for your reminder ^ ^

**Comment by Voronenko** [comment]
@vTNT lets keep issue open, perhaps you would like to submit pull request.

**Comment by vTNT** [comment]
ok

**Comment by Voronenko** [comment]
@vTNT did you try fixed routine for sem versioning on your forked code ?
